### ISSUE TITLE
error downloading thumbnail images 

### ISSUE THREAD

**Issue by stevekenney318** [nayrk/Blink#19]
Hi, I just started having an issue with the program downloading the thumbnail images.... here is the error message ->

![20220321_054819_001829](https://user-images.githubusercontent.com/53465149/159238110-26a7d3a7-df11-4e28-9452-a0e250651282.jpg)

Hopefully someone can figure it out for me.
Thanks, Steve

**Comment by stevekenney318** [comment]
Hi, Is there anyone here that can help with this ... I was able to add a couple of lines to output the variables ...

1) thumbURL = https://rest-u008.immedia-semi.com1647960212.jpg
2) thumbPath = H:\Google Drive\!!_Photos_Videos\Blink\Home\Garage\thumbnail_1647960212.jpg

Sure looks like something is wrong in $thumbURL , but I'm not sure what it's supposed to be. Is there another folder, or is it just missing a /   ?

Thanks, Steve

**Comment by stevekenney318** [comment]
Bump ... need some assistance on this.

**Comment by stevekenney318** [comment]
> Hi, I just started having an issue with the program downloading the thumbnail images.... here is the error message ->
> 
> ![20220321_054819_001829](https://user-images.githubusercontent.com/53465149/159238110-26a7d3a7-df11-4e28-9452-a0e250651282.jpg)
> 
> Hopefully someone can figure it out for me. Thanks, Steve

@nayrk , @colinreid89 , @tyuhas, @ttteee90 - Just wondering if any of you can help with this issue.

**Comment by imac211** [comment]
I'm able to download new files without a problem.  Make sure you are using the latest revision that is out there.

**Comment by stevekenney318** [comment]
> I'm able to download new files without a problem. Make sure you are using the latest revision that is out there.

You can download new thumbnails ok ?

I have a modified version I've been using, but I'll get the latest native version and try it over the weekend.

**Comment by stevekenney318** [comment]
> I'm able to download new files without a problem. Make sure you are using the latest revision that is out there.

Are you getting the thumbnails ? I got the latest version and it still doesn't work. Videos are fine, no problem. Just cant get the thumbnails... just like in the original screenshot.... looks like it's missing a / before the jpg filename. I added that and still no luck ... Then I get a 404 error , file not found. 
![image](https://user-images.githubusercontent.com/53465149/179786384-df386ac4-6b0d-44ad-b625-9dc4aabd0e7d.png)
Is no one else having this issue ?

@nayrk , @colinreid89 , @tyuhas, @ttteee90 - Just wondering if any of you can help with this ongoing issue.

Thanks, Steve

**Comment by CBR9612** [comment]
I am having this same issue with the same error. I've tried various things to update the $thumbURL but none have worked.  Sure would appreciate some help with this!

![Blink error](https://user-images.githubusercontent.com/110776830/183296897-edeafb43-1fa9-4bac-a22c-315c87ed7b41.jpg)

**Comment by stevekenney318** [comment]
> I am having this same issue with the same error. I've tried various things to update the $thumbURL but none have worked. Sure would appreciate some help with this!
> 
> ![Blink error](https://user-images.githubusercontent.com/110776830/183296897-edeafb43-1fa9-4bac-a22c-315c87ed7b41.jpg)

Finally glad to hear something else is happening issue. Not that I want anyone to have the issue. I have also tried several things to try to get that URL correct, nothing worked. I hope somebody someday with insight to were these files are located can help us

**Comment by CBR9612** [comment]
Is there some documentation somewhere on the Blink API?  I have looked and haven't found anything that talks about the thumbnails.

**Comment by ttteee90** [comment]
Looks like the API has changed a bit.
I came across this ... 

https://github.com/fronzbot/blinkpy/blob/1a75592c1b0b535d7e5346df3cbfcd077d39bbff/blinkpy/camera.py#L179 

... That references the thumbnail returns a time stamp and that appears to be the case here.

**Comment by stevekenney318** [comment]
Can you fix the program with this information ? What is this program by the
way ?

**Comment by CBR9612** [comment]
The program referenced is a Python script, and I am not a Python user so can't tell exactly, but it does mention several API changes over time. We are back to the thumbnail URL being incorrect, but I have tried multiple ways to fix it and have still been unsuccessful.  Hoping one of the original authors of this script can chime in.

**Comment by stevekenney318** [comment]
> but I have tried multiple ways to fix it

Thanks for trying ! I sure wish someone could fix it !!

**Comment by sutty550e** [comment]
I asked @nayrk a good while ago if he could help with this issue, with which I also have a problem, and he didn't get back to me. It is a nice to have only for me, and not essential. The videos still work, so I live with it as is, as I certainly don't want to pester anyone, but I thought I would post to confirm that it won't just be us few, it will be all.

Blink have a habit of changing things on their API, and this is clearly one of them. How anyone finds out where the thumbnails are now I have no idea. Sadly, it's only a matter of time before they alter the location of the videos too, and then they will also stop working.

One thing's for sure, it isn't a missing backslash, or anything else like that, because the thumbnails worked perfectly with this script for many many months.

I hope someone can chime in with a solution.

Good luck all.

Sutty

**Comment by stevekenney318** [comment]
> I hope someone can chime in with a solution.
> 
> Good luck all.
> 
> Sutty

Yes, so do I .. would love to have the thumbnails, and also not have all the error messages in the window.

-- Steve

**Comment by tejas0** [comment]
Read the information I posted in these links:

https://github.com/nayrk/Blink/issues/23#issuecomment-1450330948

https://github.com/fronzbot/blinkpy

**Comment by CBR9612** [comment]
> Read the information I posted in these links:
> 
> [#23 (comment)](https://github.com/nayrk/Blink/issues/23#issuecomment-1450330948)
> 
> https://github.com/fronzbot/blinkpy

Thank you for digging in to the doorbell video issue.  If you can figure out the thumbnail issue that would be most appreciated!

**Comment by sutty550e** [comment]
I saw you commented in the doorbell issue, and referenced back to here, so I just thought I'd nip back in to say, I'd also be really pleased to have this issue solved. I don't have a doorbell, but I'm glad that particular issue has been solved for others.

My need for the camera ID thumbnails kind of disappeared, when I deployed a newer camera, which had the photo capture option, which served my particular purpose, but still, it's a nice to have, so I'd welcome it back.

**Comment by stevekenney318** [comment]
@sutty550e - yeah, no doorbell hear either.. But nice to have all the error messages gone !!
![20230304_122917_000111_Download Blink Videos](https://user-images.githubusercontent.com/53465149/222920298-c64353c7-95e1-4377-9199-74da2f705504.jpg)
